bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -470,6 +473,7 @@ boolean complete(int bookieIndex, BookieSocketAddress host, ByteBuf buffer) {
         heardFromHostsBitSet = new BitSet(getLedgerMetadata().getEnsembleSize());
 
         readOpLogger = lh.bk.getReadOpLogger();
+        speculativeReadCounter = lh.bk.speculativeReadCounter;
```

Comment:
Better use a getter, it simplifies mocking

Reply:
Alright.